File: backend/src/main/java/de/snickit/rezeptebackend/entities/repositories/RecipeStepRepo.java
package de.snickit.rezeptebackend.entities.repositories;

import de.snickit.rezeptebackend.entities.Recipe;
import de.snickit.rezeptebackend.entities.RecipeStep;
import org.springframework.data.jpa.repository.JpaRepository;

public interface RecipeStepRepo extends JpaRepository<RecipeStep, Integer> {

}

File: backend/src/main/kotlin/de/snickit/recipe/backend/persistence/repositories/RecipeStepRepo.kt
package de.snickit.recipe.backend.persistence.repositories

import de.snickit.recipe.backend.persistence.entities.RecipeStep
import de.snickit.recipe.backend.persistence.pk.RecipeStepKey
import de.snickit.recipe.backend.persistence.repositories.template.TemplateRepository

interface RecipeStepRepo : TemplateRepository<RecipeStep?, RecipeStepKey?>
